### ISSUE TITLE
Case 3 does not work (in examples) for download.datasets(.) for multiple datasets

### ISSUE THREAD

**Issue by Harumafuji** [RevolutionAnalytics/AzureML#100]
This is a follow-on issue to issue #95.  Cases 1 and 2 (see frame1 and frame2 in function help) now work when the name argument consists of either a single dataset or multiple datasets.  Case 3; viz,

```
 # Note that one can examine all the names, sizes, etc. of the datasets
  # in ws by examining d:
  d <- datasets(ws)
  frame3 <- download.datasets(subset(d, Name == name))
```

where there is the single argument, works if name consists of a single dataset but fails if more than one are included.  Will close out #95 and isolate the remaining work to be done in this issue.

**Comment by Harumafuji** [comment]
A change has been made to the documentation indicating that the 
frame3 <- download.datasets(subset(d, Name == name))
example only currently works with a single dataset.  The problem actually lies here:
subset(d, Name == name)
ie, it seems (perhaps) the subset function has not been properly overloaded for the case where d is a Datasets object.  Will mark this issue as done and open an new issue that focuses on the subset(.) function specifically.

**Comment by andrie** [comment]
Fixed in #111
